core/application/notificationWorkflow/Types.ts

```diff
@@ -9,15 +9,30 @@ export interface NotificationAttributes {
   payload: Record<string, unknown>;
 }
 
-export interface BloodPostNotificationAttributes {
+export interface BloodDonationNotificationAttributes {
   userId: string;
   title: string;
   body: string;
   type: NotificationType;
-  payload: {
-    requestPostId: string;
-    [key: string]: unknown;
-  };
+  payload: BloodDonationPayloadAttributes;
+}
+
+export interface BloodDonationPayloadAttributes {
```

Comment:
Same for this

Reply:
i dont understand, i already separated payload type here

Reply:
Updated in 39b4b130f6a0edf11228fe5a7c3faad7d8f4dcf1